src/style.css

```diff
@@ -1,3 +1,85 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
```

Comment:
You have zeroed out the margins for `body` and `html` tags. It's not efficient for the browser to zero out your styles with `*` selector, but in your case, it's better to use `*` selector to zero out margins and paddings for all elements at once. It's called a CSS reset. This way, you ensure that all elements have the same styles across different browsers.